docs/manual/guides/install-test-versions.de.md

```diff
@@ -175,7 +172,7 @@ Version wird, sieht so aus:
         }
     },
     "extra": {
-        "contao-component-dir": "assets"
+        "contao-component-dir": "public/assets"
```

Comment:
Hm, the `contao/managed-edition` still has only `assets`:

https://github.com/contao/managed-edition/blob/3039b6e040c54081033ee330b203398773d158e9/composer.json#L30-L32

Is this a bug in the `contao/managed-edition` repository?

Reply:
Might be, the branch for 5.4 didn't exist for the managed-edition when updating this

Reply:
Also the `5.x` branch only has `assets`. Can you make a PR for `contao/managed-edition` first where we can confirm that this is actually correct?

Reply:
Also wouldn't this break for everyone updating from 4.13 or 5.3 to 5.4? 🤔

Reply:
Well, the guide is for installing the latest test version aka RC.

I did have some lovely issues updating a 5.3 demo to 5.4, would need to test it one more time.
assets doesn't get moved to public, however the setup does handle it afaik

Reply:
> Well, the guide is for installing the latest test version aka RC.

Yes, but both the `5.4` branch and `5.x` branch of the `contao/managed-edition` do not contain this change.

Reply:
https://github.com/contao/managed-edition/pull/63 <-- leaving that for documentation

Reply:
> I did have some lovely issues updating a 5.3 demo to 5.4, would need to test it one more time.

 Guess it was https://github.com/contao/contao/issues/7354 :)